packages/mockotlpserver/lib/cli.js

```diff
@@ -97,6 +99,11 @@ const OPTIONS = [
         type: 'string',
         help: `The hostname on which servers should listen, by default this is "${DEFAULT_HOSTNAME}".`,
     },
+    {
+        names: ['ui'],
+        type: 'bool',
+        help: `Start a web server to inspect traces with some charts.`,
```

Comment:
nit: don't need backtick string here... though I guess lint doesn't check that?

Reply:
it doesn't